java/flight/src/test/java/org/apache/arrow/flight/TestBasicOperation.java

```diff
@@ -99,7 +97,19 @@ public void putStream() throws Exception {
       IntVector iv = new IntVector("c1", a);
 
       VectorSchemaRoot root = VectorSchemaRoot.of(iv);
-      ClientStreamListener listener = c.startPut(FlightDescriptor.path("hello"), root);
+      ClientStreamListener listener = c.startPut(FlightDescriptor.path("hello"), root, new StreamListener<PutResult>() {
```

Comment:
Maybe we should have a default impl of StreamListener<PutResult>?

Reply:
Done.